libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFSDKOAuthClient.m

```diff
@@ -415,12 +415,16 @@ - (void)oauthCoordinatorDidCancelBrowserAuthentication:(SFOAuthCoordinator *)coo
     }
     // If no delegates implement authManagerDidCancelBrowserFlow, display Login Host List
     if (!handledByDelegate) {
+        
         SFSDKLoginHostListViewController *hostListViewController = [[SFSDKLoginHostListViewController alloc] initWithStyle:UITableViewStylePlain];
         hostListViewController.delegate = self;
+        SFSDKNavigationController *controller = [[SFSDKNavigationController alloc] initWithRootViewController:hostListViewController];
+        hostListViewController.hidesCancelButton = YES;
+    
         __weak typeof (self) weakSelf = self;
         [self.authWindow presentWindowAnimated:NO withCompletion:^{
             __strong typeof (weakSelf) strongSelf = weakSelf;
-            [strongSelf.authWindow.viewController presentViewController:hostListViewController animated:NO completion:nil];
+            [strongSelf.authWindow.viewController presentViewController:controller animated:NO completion:nil];
```

Comment:
Should we show this in multi-user scenario?

Reply:
I don't think I can change the behavior. Custom work can achieved by setting up a block in the UserAcccountManager to handle the event.